mu-plugins/blocks/global-header-footer/footer.php

```diff
@@ -13,7 +13,7 @@
 	<div class="wp-block-group global-footer__navigation-container">
 		<!-- wp:navigation {"orientation":"vertical","className":"global-footer__navigation-important","overlayMenu":"never"} -->
 			<!-- wp:navigation-link {"label":"<?php echo esc_html_x( 'About', 'Menu item title', 'wporg' ); ?>","url":"https://wordpress.org/about/","kind":"custom","isTopLevelLink":true} /-->
-			<!-- wp:navigation-link {"label":"<?php echo esc_html_x( 'Blog', 'Menu item title', 'wporg' ); ?>","url":"https://wordpress.org/news","kind":"custom","isTopLevelLink":true} /-->
+			<!-- wp:navigation-link {"label":"<?php echo esc_html_x( 'Blog', 'Menu item title', 'wporg' ); ?>","url":"https://wordpress.org/news/","kind":"custom","isTopLevelLink":true} /-->
```

Comment:
I note the Blog is called Blog in the footer, but News in the header..

The old Header had 'Blog' and it's been replaced with 'News'. 

Rosetta sites currently have 'Blog', but maybe that should also change to News.